src/components/HomepageFeatures/HomepageFeatures.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import styles from './HomepageFeatures.module.css';
+import {Button, Grid} from '@mui/material';
+import HomepageCard from '../HomepageCard/HomepageCard';
+import list from '../../config/home.json';
+import {useHistory} from '@docusaurus/router';
+
+const overrideLinkStyle = {
+  flexFlow: 'column nowrap',
+  width: '100%',
+  minHeight: '100%',
+  backgroundColor: '#fff',
+  [':hover']: {
+    boxShadow: '-1px 5px 19px 0px rgba(0,0,0,0.3)',
+    backgroundColor: '#fff'
+  }
+};
+
+const renderCards = () => {
+  const history = useHistory();
```

Comment:
Hooks should always be placed at the top of the component and not inside inner functions. Please move to `HomepageFeatures`